modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -128,3 +130,9 @@ govc vm.power -on "${VM_NAME}"
 The full syntax of the `ip=` parameter is documented in https://www.man7.org/linux/man-pages/man7/dracut.cmdline.7.html[Dracut manpages].
 
 For further information on first-boot networking, see https://coreos.github.io/afterburn/usage/initrd-network-cmdline/[Afterburn documentation].
+
+== Troubleshooting First-boot Problems
+
+You may encounter problems with your Ignition configuration that require access to the system log which appears during first-boot. To make a copy of the sytem log you can attach a serial device to the VM before booting. vSphere as well as Workstation and Fusion allow this and will save the output to a file of your choice.
+
+To attach a serial device simply modify the hardware settings of the powered off VM and add a `Serial Port`. Select the destination and name of the file to be created. Afterwards power on the VM. When encountering an error, check the file you initially specified – it should contain a copy of the system log.
```

Comment:
Not blocking this, but do you maybe know whether the serial setup can be performed directly via `govc`?

Reply:
That's a very good point. According to the `govc` [documentation](https://github.com/vmware/govmomi/blob/master/govc/USAGE.md#deviceserialadd) (also see description of `device.serial.connect`) it should work even as file backed solution. I haven't tried that yet, though. I'll update the pull request with a link to the documentation.

Reply:
Verified and fixed :)

Reply:
Ah, nice enhancement, thanks a lot!